packages/components/app/styles/components/form/masked-input.scss

```diff
@@ -36,6 +39,7 @@ $hds-form-masked-input-button-size: 24px;
   position: absolute;
   top: calc(var(--token-form-control-padding) - var(--token-form-control-border-width));
   right: calc(var(--token-form-control-padding) - var(--token-form-control-border-width));
+  grid-area: 1 / 1 / 2 / 2;
```

Comment:
Wait, is this possible, even a thing!!?!! 🤯😱🙀

(grid-area + position absolute relative to the grid area)

Reply:
when I suggested to use a grid, I was thinking of using margins for this:
```
grid-area: 1/1/2/2;
margin-top: calc(var(--token-form-control-padding) - var(--token-form-control-border-width));
margin-right: calc(var(--token-form-control-padding) - var(--token-form-control-border-width));
margin-left: auto;
```